### ISSUE TITLE
yarn.lock immutability broken by resolutions

### ISSUE THREAD

**Issue by devthejo** [rohit-gohri/yarn-lock-to-package-json#2]
Hello,

when a `package.json` is containing a `resolutions` section, the generated `package.json` try to change the `yarn.lock` on `yarn install`

**Comment by rohit-gohri** [comment]
Can you share a small example package.json? So that I can also add a test case

**Comment by devthejo** [comment]
package.json
```
{
  "name": "~dev",
  "version": "0.0.0",
  "dependencies": {
    "tiged": "^2.12.4"
  },
  "resolutions": {
    "tar": "^6.1.9"
  }
}
```

I have a fix, I will drop you a PR, and for the bin problem too

**Comment by devthejo** [comment]
here is the PR: https://github.com/rohit-gohri/yarn-lock-to-package-json/pull/4

**Comment by rohit-gohri** [comment]
Added failing test case - https://github.com/rohit-gohri/yarn-lock-to-package-json/actions/runs/4668052029/jobs/8264604371

**Comment by rohit-gohri** [comment]
Published as [v0.1.7](https://www.npmjs.com/package/yarn-lock-to-package-json/v/0.1.7). Thanks for reporting these and opening the PRs! 🎉
